src/wallet/rpcwallet.cpp

```diff
@@ -4291,7 +4367,7 @@ static RPCHelpMan send()
             // Automatically select coins, unless at least one is manually selected. Can
             // be overridden by options.add_inputs.
             coin_control.m_add_inputs = rawTx.vin.size() == 0;
-            FundTransaction(*pwallet, rawTx, fee, change_position, options, coin_control, /* override_min_fee */ false);
+            FundTransaction(*pwallet, rawTx, fee, change_position, options, coin_control, /* override_min_fee */ false, NullUniValue /* solving_data */);
```

Comment:
Nit: comment before the variable (here and elsewhere)? (Can't remember if this is preferred style)

Reply:
see https://releases.llvm.org/12.0.0/tools/clang/tools/extra/docs/clang-tidy/checks/bugprone-argument-comment.html and https://github.com/bitcoin/bitcoin/pull/21943#discussion_r717379468

Reply:
No longer relevant.